scripts/lc/install-from-dev-pkg.sh

```diff
@@ -28,7 +28,7 @@ mkdir -p $INSTALL_DIR
 
 cp -a $PWD/resources/pip_cache/. $SPHERAL_PIP_CACHE_DIR
 
-./$SCRIPT_DIR/devtools/tpl-manager.py --spack-url $SPACK_URL --init-only --spec=none --no-upstream --spheral-spack-dir $INSTALL_DIR/spheral-spack-tpls
+./$SCRIPT_DIR/devtools/tpl-manager.py --spack-url $SPACK_URL --init-only --no-upstream --spack-dir $INSTALL_DIR/spheral-spack-tpls
```

Comment:
This call to TPL manager was initially for cloning and setting up a spack repository on an air-gapped system, it didn't try to concretize or install anything. Since the new default for tpl-manager without spec is to install all dependencies what happens here now?

Reply:
The `--init-only` flag means it only clones a spack repo so it should do what we want here.